bench/benchmark.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var benchmark = require('benchmark');
+var Table = require('cli-table');
+var colors = require('colors/safe');
+
+var oldZ = require('sanctuary-type-classes');
+var newZ = require('..');
+
+
+function identity(x) {
+  return x;
+}
+
+function formatPct(pct) {
+  return ('+' + pct.toLocaleString('en', {
+    minimumIntegerDigits: 3,
+    minimumFractionDigits: 1,
+    maximumFractionDigits: 1,
+    useGrouping: false
+  })).slice(-6) + '%';
+}
+
+function format(res) {
+  return res.hz.toLocaleString('en', {maximumFractionDigits: 0}) + ' Hz '
+       + '±' + res.stats.rme.toFixed(2) + '%'
+       + '(n ' + String(res.stats.sample.length) + ')';
```

Comment:
Did you intentionally omit a space between `%` and `(`? It looks a bit cramped to me.